plugin/storage/es/spanstore/reader_test.go

```diff
@@ -276,8 +276,11 @@ func TestSpanReader_multiRead_followUp_query(t *testing.T) {
 		require.NoError(t, err)
 		sModel2, err := toDomain.SpanToDomain(&spanID2)
 		require.NoError(t, err)
-		assert.EqualValues(t, traces[0].Spans[0], sModel1)
-		assert.EqualValues(t, traces[1].Spans[0], sModel2)
+
+		for _, s := range []*model.Span{sModel1, sModel2} {
```

Comment:
This seems fine, the result can be different because the function creates traces array from a map: https://github.com/jaegertracing/jaeger/blob/a14ca3c04ca89873d21a94c2849f0337c212cb1d/plugin/storage/es/spanstore/reader.go#L378